spread.yaml

```diff
@@ -76,10 +76,9 @@ backends:
     allocate: |
       sudo tee /etc/ssh/sshd_config.d/10-spread-github-ci.conf << 'EOF'
       PasswordAuthentication yes
-      PermitEmptyPasswords yes
       EOF
 
-      sudo passwd -d runner
+      echo "runner:$SPREAD_PASSWORD" | sudo chpasswd
```

Comment:
Could not get jammy to work without a password, so using the random spread pass to ssh into localhost.